src/apsbits/demo_instrument/configs/iconfig.yml

```diff
@@ -82,5 +81,5 @@ OPHYD:
         PV_CONNECTION: *TIMEOUT
 
 # Control detail of exception traces in IPython (console and notebook).
-# Options are: Minimal, Plain, Verbose
+# Options are: 'Plain', 'Context', 'Verbose', 'Minimal', 'Docs'
```

Comment:
Is `Docs` a permitted value?  Check here: https://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-xmode

Reply:
I don't see why add so many options, just go ahead and remove 'docs'.